logos/src/logos/dbutils/dbmanager.py

```
@@ -555,6 +601,240 @@ def connect_model_provider(self, logos_key: str, model_id: int, provider_id: int
         pk = self.insert("model_provider", {"provider_id": int(provider_id), "model_id": int(model_id)})
         return {"result": f"Connected Model to Provider. ID: {pk}."}, 200
 
+    def add_model_provider_config(
+        self,
+        logos_key: str,
+        model_id: int,
+        provider_name: str,
+        cold_start_threshold_ms: float = None,
```

Ruff (0.14.6)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)